telemetry-schema.md

```diff
@@ -0,0 +1,71 @@
+## Attributes
+
+Key | Description
+--- | ---
+event | `perf-report` (can this take other values?)
+results | String holding a JSON-encoded `OuterResults`
+sentCount | string-encoded integer representing the number of `Results` in each `Test`
+
+## OuterResults
+
+Key | Description
+--- | ---
+status | `"complete"` (Can this take other values?)
+tests | array of `Test`
+
+## Test
+
+Key | Description
+--- | ---
+ccheck | ?
+doh | `true` or `false`; whether DOH was used to resolve the remote endpoint
+facebook | `true` or missing; whether the remote endpoint is Facebook
+label | friendly slug with a 1:1 mapping to a (`doh`, `url`) pair
+results | array of `Result`
+url | URL of endpoint
```

Comment:
I'm dropping this key too for network/storage improvements. I can add it back in easily if you want it for convenience. Again can be reverse engineered from the label.